Atomic/atomic.py

```diff
@@ -696,6 +820,407 @@ def install(self):
         if not self.args.display:
             return util.check_call(cmd)
 
+    def _system_images(self):
+        repo = self._get_ostree_repo()
+
+        revs = [x for x in repo.list_refs()[1] if x.startswith(OSTREE_OCIIMAGE_PREFIX) and len(x) != len(OSTREE_OCIIMAGE_PREFIX) + 64]
```

Comment:
Might want to spread this across multiple lines. https://www.python.org/dev/peps/pep-0008/#maximum-line-length